src/content/en/2020/markup.md

```diff
@@ -626,34 +704,35 @@ Using `target="_blank"` is known as a [security vulnerability](https://mathiasby
 <figcaption>{{ figure_link(caption="Blank relationships.", sheets_gid="1876528165", sql_file="pages_wpt_bodies_by_device.sql") }}</figcaption>
 </figure>
 
-As a rule of thumb—also [for usability reasons](https://www.nngroup.com/articles/new-browser-windows-and-tabs/)—, prefer not to use `target=_blank`. 
+As a rule of thumb and for [usability reasons](https://www.nngroup.com/articles/new-browser-windows-and-tabs/), prefer not to use `target="_blank"`. 
 
 <p class="note">Within the latest Safari and Firefox versions, setting <code>target="_blank"</code> on <code>a</code> elements implicitly provides the same <code>rel</code> behavior as setting <code>rel="noopener"</code>. This is currently being <a href="https://bugs.chromium.org/p/chromium/issues/detail?id=898942">implemented in Chromium</a> as well.</p>
 
+{# TODO(editors, authors): The following two sections each read like conclusions. Consider merging. #}
 ## Status and trends
 
-We’ve sprinkled some observations throughout the chapter, and you’ll have made your own observations. At the end of this 2020 analysis, here are some things that stood out for us.
+We've sprinkled some observations throughout the chapter, and you'll have made your own observations. At the end of this 2020 analysis, here are some things that stood out for us.
 
-Fewer pages land in quirks mode. In 2016, that number was at [around 7.4%](https://discuss.httparchive.org/t/how-many-and-which-pages-are-in-quirks-mode/777), at the end of 2019, we observed [4.85%](https://twitter.com/zcorpan/status/1205242913908838400), and now we’re at about 3.97%. This trend, to quote Simon Pieters in his review of this chapter, seems indeed clear and encouraging.
+{# TODO(authors): Where does 3.97% come from? Can we link to it? #}
```

Comment:
See doctypes section—however, on checking, I believe it’s 3.01% (100% – 96.99%). @zcorpan, am I missing something?

Reply:
See analysis in https://docs.google.com/document/d/1m1YvDqWgZfqmcmAEkCnXncIJwsM1QCu6Rv7aPWjG1Ms/edit?usp=drivesdk&disco=AAAAIwTxu-k

Short version: some doctypes trigger quirks mode, so you need to count them, too.

Reply:
This is worth calling out in a standalone "big number" figure if significant. That will also provide a way to give more context and link it back to the data.

I see in the linked comment thread that the number was arrived at by copying the data to a new spreadsheet for analysis. I'd encourage you to keep all analysis in the public doc so it's easily discoverable. @zcorpan do you have edit access to do the same math in the official results sheet? (If not, request access through Sheets and I'll grant it shortly)

Reply:
Done: https://docs.google.com/spreadsheets/d/1Ta7amoUeaL4pILhWzH-SCzMX9PsZeb1x_mwrX2C4eY8/edit?ts=5f4e700c&pli=1#gid=1981441894&range=F102

Reply:
Thanks, added a big number figure and referenced the sheet.